.github/workflows/main.yml

```
@@ -15,42 +15,42 @@ env:
+  tests:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
 
-      #- name: Run all tests
-      #  run: cargo test --verbose -- --ignored
-      #  env:
-      #    RUST_BACKTRACE: 1
+      - name: Get cargo
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.4)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)